app/controllers/certificates_controller.rb

```diff
@@ -0,0 +1,81 @@
+class CertificatesController < ApplicationController
+  before_action :authorise_manage_certificates
+  before_action :authorise_manage_current_certificate, only: %i[show edit destroy]
+
+  def index
+    @pagy, @certificates = pagy(Certificate.where(organisation: current_organisation))
```

Comment:
Is pagy really necessary here? I don't think there'll be that many certificates right?

Reply:
i guess not - i think there will end up being a few iterations of the gui, so can probably stay until next time...